android/sdk/src/main/java/org/jitsi/meet/sdk/audiomode/AudioModeModule.java

```diff
@@ -300,7 +300,6 @@ private boolean updateAudioRoute(int mode) {
             audioManager.setMode(AudioManager.MODE_NORMAL);
             audioManager.abandonAudioFocus(null);
             audioManager.setSpeakerphoneOn(false);
-            audioManager.setMicrophoneMute(true);
```

Comment:
No, I'm not insisting - it was just a thought. I was just thinking that if the line is gone it's not easy to find the change in potentially long commits history. But if you have a line you can look up the commit that changed it.